srcpkgs/boost/template

```diff
@@ -31,24 +32,19 @@ replaces="libboost_program_options1.69>=0 libboost_math_c99f1.69>=0
  libboost_wserialization1.69>=0 libboost_context1.69>=0
  libboost_stacktrace_addr2line1.69>=0"
 
-subpackages="libboost_program_options1.72 libboost_math_c99f1.72
- libboost_stacktrace_basic1.72 libboost_stacktrace_noop1.72
- libboost_iostreams1.72 libboost_math_tr1l1.72 libboost_filesystem1.72
- libboost_chrono1.72 libboost_math_tr1f1.72 libboost_wave1.72
- libboost_graph1.72 libboost_locale1.72 libboost_regex1.72
- libboost_fiber1.72 libboost_type_erasure1.72 libboost_coroutine1.72
- libboost_atomic1.72 libboost_prg_exec_monitor1.72 libboost_math_c991.72
- boost-python3-1.72 libboost_log_setup1.72 boost-devel
- libboost_timer1.72 libboost_system1.72 libboost_unit_test_framework1.72
- libboost_serialization1.72 libboost_math_tr11.72 boost-build
- libboost_math_c99l1.72 boost-python1.72 libboost_contract1.72
- libboost_date_time1.72 libboost_log1.72 libboost_random1.72
- libboost_thread1.72 libboost_container1.72 boost-jam
- libboost_wserialization1.72 libboost_context1.72"
-
-if [ -z "$CROSS_BUILD" ]; then
-	subpackages+=" libboost_stacktrace_addr2line1.72"
-fi
+subpackages="libboost_program_options1.79 libboost_math_c99f1.79
+ libboost_iostreams1.79 libboost_math_tr1l1.79 libboost_filesystem1.79
+ libboost_chrono1.79 libboost_math_tr1f1.79 libboost_wave1.79
+ libboost_graph1.79 libboost_locale1.79 libboost_regex1.79
+ libboost_type_erasure1.79 libboost_coroutine1.79
+ libboost_atomic1.79 libboost_prg_exec_monitor1.79 libboost_math_c991.79
+ boost-python3-1.79 libboost_log_setup1.79 boost-devel
+ libboost_timer1.79 libboost_system1.79 libboost_unit_test_framework1.79
+ libboost_serialization1.79 libboost_math_tr11.79 boost-build
+ libboost_math_c99l1.79 boost-python1.79 libboost_contract1.79
+ libboost_date_time1.79 libboost_log1.79 libboost_random1.79
+ libboost_thread1.79 libboost_container1.79 boost-jam
+ libboost_wserialization1.79 libboost_context1.79"
```

Comment:
If you're dropping the no-cross `libboost_stacktrace_addr2line1.72`,  you should be able to just drop the entier `subpackages` declaration.

Reply:
subpackages is currently abused  in `boost-devel` as a list of deps, so you would have to move the list of packages to there.